kai/reactive_codeplanner/task_runner/compiler/maven_validator.py

```diff
@@ -91,6 +92,12 @@ def from_match(
     def compiler_error_message(self) -> str:
         return self.message
 
+    def validate_filepath(self) -> None:
```

Comment:
Do we call this every time we create one of these? If so, would it make sense to put this in the `dataclass`'s [`__post_init__` function](https://docs.python.org/3/library/dataclasses.html#dataclasses.__post_init__)?

Reply:
+1 we should for sure do that!
f